enterprise/frontend/src/embedding-sdk/lib/interactive-question/run-question-query.ts

```diff
@@ -35,6 +36,10 @@ export async function runQuestionQuerySdk(
       ignoreCache: false,
       isDirty: isQueryDirty,
     });
+
+    const [{ data }] = queryResults;
+    const sensibleDisplays = getSensibleDisplays(data);
+    question = question.maybeResetDisplay(data, sensibleDisplays, undefined);
```

Comment:
use similar function call from the internal use case
https://github.com/metabase/metabase/blob/1f19dee6c51bf07160d64426891294f2aab92f74/frontend/src/metabase/query_builder/actions/querying.ts#L222-L226

We don't need to pass the 3rd argument which is `previousSensibleDisplays` from a previously run query because 2 reasons:
1. we want to prioritize the first render of the ad-hoc question
2. There would be a lot of change just to get the previously run `queryResults` just so we can calculate this `previousSensibleDisplays` since this redux action is called from multiple places.